Tools/AP_Periph/can.cpp

```diff
@@ -1708,6 +1709,36 @@ void AP_Periph_FW::can_gps_update(void)
                         &buffer[0],
                         total_size);
     }
+
+    // send the gnss tatus packet
+    {
+        ardupilot_gnss_Status status {};
+
+        status.healthy = gps.is_healthy();
+        if (gps.logging_present() && gps.logging_enabled() && !gps.logging_failed()) {
+            status.status |= ARDUPILOT_GNSS_STATUS_STATUS_LOGGING;
+        }
+        uint8_t idx; // unused
+        if (!gps.first_unconfigured_gps(idx)) {
```

Comment:
should this only be set if healthy?

Reply:
Not being healthy triggers a different check in ArduPilot already, but it's not a bad idea...